### PR TITLE
Add Chinese translation for global configuration

### PR DESCRIPTION
**Pull request by LinuxSuRen**
@oleg-nenashev

### CODE REVIEW

File: core/src/main/resources/hudson/model/UsageStatistics/global_zh_CN.properties
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/UsageStatistics/global_zh_CN.properties]: Wrong copyright.

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: While the followup change to https://github.com/jenkinsci/jenkins/pull/3287 is not filed, this will remove existing translations without replacement.

**Review by daniel-beck — COMMENTED** [review]
Should not be merged before existing translations have been migrated to the `f:apply` control as discussed in https://github.com/jenkinsci/jenkins/pull/3287.

File: core/src/main/resources/jenkins/model/Jenkins/configure.jelly
Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: #3287 already been merged.

Comment (daniel-beck) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: Yes, without moving the existing localizations to the `f:apply` control. It's similar to the discussion in https://github.com/jenkinsci/jenkins/pull/3319.

If we merge this change, system configuration will lose the apply button localization for Bulgarian, German, French, Italian, etc., as there are no corresponding translations to https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/apply.jelly -- only English and Chinese.

Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: It's my mistake. So your option is put them back, or remove control to `f:apply`.

Comment (daniel-beck) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: In my opinion, the preferable option --as in #3319 -- would be to just move the translations to `apply_*.properties` files.

If you don't wish to work on translations of languages you don't know, keeping the translations around and reverting part of this PR would also work.

Comment (LinuxSuRen) [reply core/src/main/resources/jenkins/model/Jenkins/configure.jelly]: Ok than, I move those files.

**Comment by oleg-nenashev** [comment]
We will re-review it today

**Review by oleg-nenashev — APPROVED** [review]

**Comment by daniel-beck** [comment]
Will need to re-review the localization situation after the linked PR was merged.

**Comment by LinuxSuRen** [comment]
Does something block this PR merge into master?
@daniel-beck

**Review by daniel-beck — APPROVED** [review]
Applys seem to have been restored.

**Pull request merged**
